### DISCUSSION TITLE
Bug: difficult to download iOS build from build dashboard

### DISCUSSION THREAD

**Discussion by bolandrm** [codemagic-ci-cd/codemagic-docs#1877] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

1.  Create a build dashboard
2. View the build dashboard on iOS
3. Attempt to download a build

### Expected results

The download should begin.

### Actual results

The build/download card closes and the link is not actually pressed.  So the build doesn't begin.

I recorded a video of the bug:  https://imgur.com/a/nR8xsYK

The video includes sound/audio narration.

**Comment by icarusdust** [comment]
Hi @bolandrm, thanks for reporting it, I'll keep you posted about the update soon.

**Comment by icarusdust** [comment]
Hi @bolandrm, it looks fine now, please try it again and let me know if you have any further questions.

**Comment by bolandrm** [comment]
@icarusdust I'm still experiencing the same behavior that is shown in the video.

**Reply by icarusdust** [reply]
Please try cleaning caches from your device or go incognito to try it, it works alright for me at the moment.

**Reply by bolandrm** [reply]
- I cleared website data for codemagic.io
- I entered safari private browsing mode
- I'm on Safari on iOS 16.5.1

No changes.  Here's another video of the current behavior:  https://imgur.com/a/ZkQ7oJ5

**Comment by bolandrm** [comment]
@icarusdust Why was this closed?  Myself and my colleagues are still experiencing this issue.

**Reply by icarusdust** [reply]
@bolandrm I think it got closed when I tested it and worked for me. I'll get back to you asap.
